### DISCUSSION TITLE
Email when you have messages

### DISCUSSION THREAD

**Discussion by Oren-Lindsey** [ScratchAddons/ScratchAddons#1954] [Ideas]
Can you add something to scratch addons that sends you an email whenever you get a new message? It would be so useful. If you have an apple watch like me or a phone that alerts you when you get an email, you could be notified the instant someone comments on or likes on of your projects. You wouldn't have to check your computer all the time.

**Comment by Endermaster3690** [comment]
sounds cool but we'd have to enter an email address so people would not be comfortable activating the addon.

**Reply by Oren-Lindsey** [reply]
I would. You could have it turned off by default. As long as its private I think the kind of people who use this add-on would be ok with it

**Reply by Oren-Lindsey** [reply]
Could you make it something that runs locally?

**Reply by Oren-Lindsey** [reply]
Then it would be private

**Reply by Oren-Lindsey** [reply]
And even if you couldn't, you don't have to enable it. I just think there actually are a lot of people who would use this.

**Reply by FunctionalMetatable** [reply]
This needs a server to operate. Its practically no use unless someone with a server is willing to let SA use it.

**Reply by Endermaster3690** [reply]
> This needs a server to operate. Its practically no use unless someone with a server is willing to let SA use it.

yeah

> Could you make it something that runs locally?

and i am not a scripter.

**Comment by retronbv** [comment]
This would require an external server, this is most likely going to be rejected

**Reply by Oren-Lindsey** [reply]
couldn't you use AWS or something?

**Reply by retronbv** [reply]
Either way how would the server check you have messages? We would probably need to store passwords on the server which is insecure

**Reply by Oren-Lindsey** [reply]
This would be really easy if we had an app for it. Its really hard because we don't have an app, we have an extension.

**Reply by retronbv** [reply]
> This would be really easy if we had an app for it. Its really hard because we don't have an app, we have an extension.

Have fun trying to convert SA to an app

**Reply by WorldLanguages** [reply]
> couldn't you use AWS or something?

Sure, as long as you pay for the bills...

**Reply by FunctionalMetatable** [reply]
> This would be really easy if we had an app for it. Its really hard because we don't have an app, we have an extension.

try using https://itchy-ionic.vercel.app then

**Reply by GrahamSH-LLK** [reply]
> This would be really easy if we had an app for it. Its really hard because we don't have an app, we have an extension.

?? Apps still can't send emails

**Comment by GrahamSH-LLK** [comment]
Yeah, no. This is a cool idea _in theory_, but it falls apart if you think about it. To send messages, we'd need A. a server, B. to pay for an email service and set it up.
